tests/common.go

```diff
@@ -6353,6 +6354,58 @@ func GetPoolExpansionEligibility(stNode *node.Node) (map[string]bool, error) {
 	return eligibilityMap, nil
 }
 
+// GetPoolMaxCloudDriveLimit identifying the max drives allowed based on type of setup
+func GetPoolMaxCloudDriveLimit(stNode *node.Node) (int32, error) {
+	var err error
+
+	namespace, err := Inst().V.GetVolumeDriverNamespace()
+	if err != nil {
+		return 0, err
+	}
+
+	var maxCloudDrives int32
+
+	if _, err := core.Instance().GetSecret(PX_VSPHERE_SCERET_NAME, namespace); err == nil {
+		maxCloudDrives = VSPHERE_MAX_CLOUD_DRIVES
+	} else if _, err := core.Instance().GetSecret(PX_PURE_SECRET_NAME, namespace); err == nil {
+		maxCloudDrives = FA_MAX_CLOUD_DRIVES
+	} else {
+		maxCloudDrives = CLOUD_PROVIDER_MAX_CLOUD_DRIVES
+	}
+
+	if err != nil {
+		return 0, err
+	}
+	return maxCloudDrives, nil
+}
+
+func GetPoolsinNode(stNode *node.Node) (map[int]string, []int) {
```

Comment:
You can get pools from existing node object, this is not required

Reply:
removed